dbmodel/database script.sql

```diff
@@ -2,7 +2,9 @@
 CREATE TABLE IF NOT EXISTS content (
   id varchar(255) PRIMARY KEY NOT NULL,
   encryption_key varchar(64) NOT NULL,
-  location text NOT NULL
+  location text NOT NULL,
+  length int(11) NOT NULL,
```

Comment:
`CREATE TABLE IF NOT EXISTS` in `index/index.go` uses `length bigint`, so `int(11)` should be consistent. I am okay with `bigint`. @RemiBauzac ?